plugins/@grouparoo/clickhouse/src/lib/export/connection.ts

```diff
@@ -0,0 +1,55 @@
+import { PluginConnection } from "@grouparoo/app-templates/dist/source/table";
+
+import { destinationOptions } from "@grouparoo/mysql/dist/lib/export/destinationOptions";
+import { destinationMappingOptions } from "@grouparoo/mysql/dist/lib/export/destinationMappingOptions";
+import { exportArrayProperties } from "@grouparoo/mysql/dist/lib/export/exportArrayProperties";
+
+import { exportRecord } from "./exportRecord";
+
+export const getConnection = (): PluginConnection => ({
+  name: "clickhouse-export-records",
+  displayName: "ClickHouse Export Record",
+  direction: "export",
+  description:
+    "Export Records to a ClickHouse table.  Groups will be exported to a secondary table linked by a foreign key.",
+  app: "clickhouse",
```

Comment:
Once you rebase from `main`, this will become 

```suggestion
  apps: ["clickhouse"],
```